### PR TITLE
Run prettier and fix eslint errors

### PR DESCRIPTION
**Pull request by ra100**
- src folder is run through prettier
- eslint rules are fixed where possible, some were not, as mutating parameter

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -7,6 +7,23 @@ plugins:
...
+  NSMakeRect: true
+  NSMakeSize: true
+  NSMatrix: true
+  NSRadioButton: true
+  NSRadioModeMatrix: true
+  NSString: true
+  NSURL: true
+  NSURLSession: true
+  NSUserDefaults: true
+  NSUUID: true
```
Comment (eduardogr) [inline .eslintrc.yml:26]: Why is all of this??

File: src/analytics.js [hunk 2]
```diff
@@ -1,41 +1,48 @@
...
   // Event value
   if (value) {
-    url += "&ev=" + encodeURI(value);
+    url += `&ev=${encodeURI(value)}`
   }
 
-  var session = NSURLSession.sharedSession(),
-  task = session.dataTaskWithURL(NSURL.URLWithString(NSString.stringWithString(url)));
-  task.resume();
+  const session = NSURLSession.sharedSession();
```
Comment (eduardogr) [inline src/analytics.js:43]: @ra100 have tried this version of Camilo in sketch??? 

@Pausansol

**Review by eduardogr — COMMENTED** [review]
Have you try this version in sketch? i am seeing that you are updating the ES version, i'd like you to verify this since sketch is a little bit different @Pausansol

File: .eslintrc.yml (hunk 1 above)
Comment (ra100) [reply .eslintrc.yml:26]: withou this, eslint is complaining, that we are using "undefined" variable, as it's global

File: src/analytics.js (hunk 2 above)
Comment (ra100) [reply src/analytics.js:43]: I still need to try it all. But my assumption is, that it's anyway transpiled to ES5 by `npm run build`

Comment (ra100) [reply src/analytics.js:43]: @eduardogr so yes, it works in Sketch.

**Pull request merged**